commons/src/test/java/io/aiven/kafka/tieredstorage/commons/UniversalRemoteStorageManagerConfigTest.java

```diff
@@ -40,6 +42,8 @@ void minimalConfig() {
             )
         );
         assertThat(config.objectStorageFactory()).isInstanceOf(ObjectStorageFactory.class);
+        assertThat(config.segmentManifestCacheSize()).isEqualTo(Optional.of(1000L));
```

Comment:
Nit(here and in other places):
```suggestion
        assertThat(config.segmentManifestCacheSize()).hasValue(1000);
```

Reply:
Great, fixed